p2p/mock/appStatusHandlerMock.go

```diff
@@ -66,7 +66,9 @@ func (ashm *AppStatusHandlerMock) Decrement(key string) {
 		return
 	}
 
-	ashm.data[key] = vUint64 - 1
+	if ashm.data[key] != 0 {
```

Comment:
I think that the correct condition should be -> if vUint64 != 0

Reply:
you're right. fixed